File: Taller 08/Python/persona.py
class Persona:
    def __init__(self, nombre, edad, altura, sexo):
        self.__nombre = nombre
        self.__edad = edad
        self.__altura = altura
        self.__sexo = sexo

    def getNombre(self):
        return self.__nombre

    def set_nombre(self, nombre):
        self.__nombre = nombre

    def getEdad(self):
        return self.__edad

    def set_edad(self, edad):
        self.__edad = edad

    def getAltura(self):
        return self.__altura

    def set_altura(self, altura):
        self.__altura = altura

    def getSexo(self):
        return self.__sexo

    def set_sexo(self, sexo):
        self.__sexo = sexo

File: Taller 04/Java/src/main/java/classroom/Persona.java
package classroom;

public class Persona {

    long cedula = 1;
    String nombre;
    static int totalPersonas;
    
    static {
        totalPersonas = 0;
    }

    public Persona(long cedula, String nombre) {
        this.cedula = cedula;
        this.nombre = nombre;
        totalPersonas++;
    }
    
    public Persona(String nombre, long cedula) {
        this.cedula = cedula;
        this.nombre = nombre;
        totalPersonas++;
    }

    public Persona(long cedula) {
        this.cedula = cedula;
        this.nombre = "";
        totalPersonas++;
    }

    public Persona(String nombre) {
        this.nombre = nombre;
        totalPersonas++;
    }
    
    public Persona() {
    	this.nombre = "";
    	this.cedula = 0;
    	totalPersonas++;
    }
    
    public long getCedula() {
        return cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}
